homeassistant/components/sensor/wsdot.py

```diff
@@ -0,0 +1,145 @@
+"""
+Support for Washington State Department of Transportation (WSDOT) data.
+
+Data provided by WSDOT is documented at http://wsdot.com/traffic/api/
+"""
+import logging
+import re
+from datetime import datetime, timezone, timedelta
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY, CONF_NAME, ATTR_ATTRIBUTION,
+    )
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_TRAVEL_TIMES = 'travel_time'
+CONF_TRAVEL_TIME_ID = 'id'
+CONF_ATTRIBUTION = "Data provided by WSDOT"
```

Comment:
This might be a nit too small to pick, but usually CONF_ constants define configuration key names. It might be better if this variable were named something like `ATTRIBUTION`